### ISSUE TITLE
Enhancement: Colorize Main Menu & Story Menu

### ISSUE THREAD

**Issue by JOELwindows7** [Perkedel/Kaded-fnf-mods#27]
## What is your suggestion, and why should it be implemented?
Colorize menu in main menu & Story Menu just like
- https://gamebanana.com/mods/322585 Main Menu
- https://gamebanana.com/mods/322581 Story Menu
but procedural. colorize when selected. uncolorize when unselected.

No, no need to download the mod itself. recreate the functionality of it!

Simple! most week menu are white stroke. if the week banner contains other than white just colorize the selection still white to prevent distort.

now for main menu, same! colorize only if selected. when unselected, uncolorize back to former.

basically another color information for week banner if selected (default white?). maybe also if unselected (default white)

Because to make cool. let's make this place fancier than ever before!

**Comment by JOELwindows7** [comment]
# yeyeyeyeyeyeyeyeye

- The main menu has been colored. When select, it tints to respective color, and turns back to default tint when deselected. tweening!
- We could not color week menu, because banner images already has color on its own, so tinting would darken them. Instead, it's the flash blink when selected we color.
